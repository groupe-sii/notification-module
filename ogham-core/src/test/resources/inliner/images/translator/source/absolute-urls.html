<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
</head>
<body>
	<img src="images/h1.gif" />
	<img src="classpath:/inliner/images/translator/source/images/fb.gif" />
	<img src="http://absolute.gif" />
</body>
</html>