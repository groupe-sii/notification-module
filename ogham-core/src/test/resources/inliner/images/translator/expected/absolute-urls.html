<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
 <head> 
 </head> 
 <body> 
  <img src="cid:0"> 
  <img src="cid:1"> 
  <img src="http://absolute.gif">  
 </body>
</html>